src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataService.java

```diff
@@ -128,7 +128,7 @@ public CaseData getCase(UUID caseUUID) {
         return caseData;
     }
 
-    private CaseData getCaseData(UUID caseUUID) {
+    public CaseData getCaseData(UUID caseUUID) {
```

Comment:
Wasn't this changed to `protected` in a inflight draft PR?

Reply:
👍 ⚡